.golangci.yml

```diff
@@ -0,0 +1,20 @@
+linters:
+  enable:
+    - nakedret 
+    - interfacer
+    - varcheck
+    - deadcode 
+    - structcheck 
+    - misspell
+    - maligned 
+    - ineffassign
+    - goconst
+    - goimports
+    - errcheck 
+    - dupl 
+    - unparam 
+    - golint 
+    - staticcheck
+    - unused
+    - gosimple
+    
```

Comment:
Nit ; 
<img width="232" alt="Screen Shot 2020-08-10 at 16 27 26" src="https://user-images.githubusercontent.com/7708031/89811891-39586500-db37-11ea-8c07-f41afe457eb3.png">

Reply:
You need to remove 1 space.

Reply:
it is unclear what the "nit" is... I took an earlier review comment that you wanted a newline... which is there now.   What specifically would you like resolved?

Reply:
oops... I'm getting used to a new editor and it provided a space / tab... nit makes sense.  apologies